src/AccessibilityInsights.Extensions.AzureDevOps/Models/IssueResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using System;
+
+namespace AccessibilityInsights.Extensions.AzureDevOps.Models
+{
+    public class IssueResult : IIssueResult
+    {
+        public string DisplayText { get; set; }
```

Comment:
>set [](start = 41, length = 3)

Should DisplayText and IssueLink be get-only? I can't think of any case where the app would need to change them. The implementing class can have setters even if the abstraction doesn't

Reply:
They are get only in the abstraction; this is the impementation.